php/Exchange.php

```diff
@@ -1167,6 +1177,13 @@ public function __construct($options = array()) {
         //     }
         // }
 
+        $this->set_exchange_prop_all_case('proxyUrl', null);
```

Comment:
@ttodua we already declare and initialize the props (78-87), why do we need to do this?